bin/ttmscalr

```diff
@@ -1201,4 +1202,33 @@ Main {
       url_shard(replacement)
     end
   end
+
+  mode 'psql:tunnel' do
+    description 'Open an SSH tunnel to an RDS DB'
```

Comment:
This might be a bit confusing.  It's not really opening it to an RDS DB, it i opening it to a debug role on a farm